src/components/Badges/Badges.tsx

```diff
@@ -0,0 +1,42 @@
+import { colors } from '../../typography/colors';
+import { Caption } from '../typography/caption/caption';
+import './index.css';
+import { ReactNode, useState } from 'react';
+
+export interface PropsBadges {
```

Comment:
```suggestion
export interface BadgeProps {
```

Convenção de nomenclatura é o nome do componente + `Props`.